source/delegate/contracts/DelegateV2.sol

```diff
@@ -269,6 +320,13 @@ contract DelegateV2 is IDelegateV2, Ownable {
     if (remainingSenderAmount > 0) return 0;
   }
 
+  /**
+   * @notice Get the signer-side quote for a trade
+   * @param signerAmount uint256 The amount of tokens they want to send
```

Comment:
Might as well use "supply" for each side, to avoid confusion with "sender" language.
For example, here would be "...amount of tokens the user will supply"

Reply:
Yep I like that